server/storage/mvcc/key_index_test.go

```diff
@@ -453,6 +474,12 @@ func TestKeyIndexCompactAndKeep(t *testing.T) {
 		},
 	}
 
+	tombstones := map[int64]struct{}{
+		6:  {},
+		12: {},
+		16: {},
+	}
```

Comment:
You should be able to check whether a compacted revision is a tombstone programatically.
```suggestion
```

Reply:
Hard coding the tombstone is weird and not friendly to other contributors, and according hard to maintain.

Reply:
Updated.